spec/enumerables_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+# spec/enumerables_spec.rb
+
+require_relative '../main.rb'
```

Comment:
I think we have required the file wrongly. Let's change `require_relative '../main.rb'` to `require_relative '../bin/enumerable.rb'`.

Reply:
Done, thank you so much